manifests/cr-instance-example.yaml

```
@@ -0,0 +1,43 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+attributes:
+  Name: "topologyManagerPolicy"
```

nit: should we follow lowercase (or camelcase) for the field names?
```suggestion
  name: "topologyManagerPolicy"
```

---

Sure, this is just an example. Typically the CR instance would be populated by the exporter component and is expected to have the fields in lowercase. I will update the example here.